homework1.6

avvasil opened the pull request.

avvasil committed:
homework1.6

OKrylov commented on src/pro/sky/Main.java:
```diff
@@ -3,6 +3,55 @@
+        int minSpendingPerDay = 200001;
+        int maxSpendingPerDay = 99999;
+
+        for (int i = 0; i <= arr.length - 1; i++) {
```
лучше сокращать до 
```suggestion
        for (int i = 0; i < arr.length; i++) {
```

OKrylov commented on src/pro/sky/Main.java:
```diff
@@ -3,6 +3,55 @@
+        }
+    }
+
+    static int[] arr = generateRandomArray();
```
поля следует объявлять вверху тела класса, перед всеми методами

OKrylov reviewed (approved):
Отличная работа! ДЗ реализовано корректно, особых замечаний по коду не имею. Продолжайте в том же духе!

avvasil committed:
Update src/pro/sky/Main.java

better to reduce

for (int i = 0; i <= arr.length - 1; i++) {

for (int i = 0; i < arr.length; i++) {

Co-authored-by: OKrylov <krylov92oa@gmail.com>

avvasil committed:
homework1.6
